extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibProcessor.java

```diff
@@ -145,8 +147,9 @@ public void buildFromNative(ContainerImageConfig containerImageConfig, JibConfig
             List<ContainerImageLabelBuildItem> containerImageLabels,
             BuildProducer<ArtifactResultBuildItem> artifactResultProducer) {
 
-        if (!containerImageConfig.build && !containerImageConfig.push && !buildRequest.isPresent()
-                && !pushRequest.isPresent()) {
+        Boolean buildContainerImage = containerImageConfig.build || buildRequest.isPresent();
```

Comment:
```suggestion
        boolean buildContainerImage = containerImageConfig.build || buildRequest.isPresent();
```

Reply:
I wanted to use toString.

Obviously it could be done with a string concat, but I think it doesn't matter 🙂